web/business-registry-dashboard/app/app.config.ts

```diff
@@ -111,40 +110,6 @@ export default defineAppConfig({
     notifications: {
       position: 'bottom-0 left-1/2 -translate-x-1/2'
     },
-    pagination: {
```

Comment:
This UI doesn't use a NuxtUI UPagination component so we don't need to provide local config for it.

If needed "globally", this config should be added to the core layer: https://github.com/bcgov/sbc-connect-common/blob/main/nuxt/layers/core/app/app.config.ts.